cli/darwin-release.yml

```diff
@@ -0,0 +1,28 @@
+project_name: turbo
+before:
+  hooks:
+    - make compile-protos
+    - go mod tidy
+
+dist: dist-darwin
+
+builds:
+  - id: turbo
+    main: ./cmd/turbo
+    binary: bin/turbo
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.CommitDate}} -X main.builtBy=goreleaser
+    mod_timestamp: "{{ .CommitTimestamp }}"
+    env:
+      - CGO_ENABLED=1
+    targets:
+      - darwin_arm64
+      - darwin_amd64
+
+archives:
+  - format: binary
+
+release:
+  disable: true
```

Comment:
How are these `cli/*-release.yml` files being used? I don't see them in the workflows

Reply:
They are config files for `goreleaser`, referenced in the `Makefile`